test_integration/shared.js

```diff
@@ -54,6 +54,7 @@ class Chain {
         this.deployerAddress = '';
         this.organizationAddress = '';
         this.networkId = '*';
+        this.accounts = [];
```

Comment:
Please also update the JSDoc.

I still think one can use web3 for the accounts, but this is also fine.
With truffle, you are not really supposed to use web3 directly, so they provide the accounts to you.